doc/Readme_For_Setting_Up_New_Features

```diff
@@ -0,0 +1,6 @@
+When setting up new feature please make sure the feature is activated in seed.sb file.
```

Comment:
typo - should be `seeds.rb` and I would make it specifically `db/seeds.rb` to help people to know where to find it.

We should also mention that they should create a migration, and link to relevant documentation on Feature flags, e.g. http://martinfowler.com/bliki/FeatureToggle.html